language/e2e_tests/src/tests/mint.rs

```diff
@@ -67,7 +68,7 @@ fn mint_to_new_account() {
     // create and publish a sender with 1_000_000 coins
```

Comment:
This comment doesn't match either the old code or the new code. Please delete it.